src/site-server.ts

```diff
@@ -51,6 +51,7 @@ export async function createSiteWorkingDirectory(
 
 	await purgeWpConfig( wpVersion );
 	await recursiveCopyDirectory( getWordPressVersionPath( wpVersion ), path );
+	await recursiveCopyDirectory( 'vendor/adminer', path + '/adminer' );
```

Comment:
Might need an adminer.config.php that defines some site constants. Mainly to populate Adminer's header/breadcrumb with the site name for the sake of differentiation.